packages/block-library/src/post-tags/edit.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+function PostTagsDisplay() {
+	const [ tags ] = useEntityProp( 'postType', 'post', 'tags' );
+	const tagLinks = useSelect(
+		( select ) => {
+			const { getEntityRecord } = select( 'core' );
+			let loaded = true;
+			const links = tags.map( ( tagId ) => {
+				const tag = getEntityRecord( 'taxonomy', 'post_tag', tagId );
+				if ( ! tag ) {
+					return ( loaded = false );
+				}
+				return (
+					<a key={ tagId } href={ tag.link }>
```

Comment:
Sorry for my ignorance -- do we have existing UI patterns to make these links functional in editing? Could otherwise be a bit confusing to users.

Reply:
They are emerging: https://github.com/WordPress/gutenberg/issues/19204